pkg/templates/kubeadm/v1beta3/kubeadm.go

```diff
@@ -52,7 +52,7 @@ const (
 )
 
 // NewConfig returns all required configs to init a cluster via a set of v1beta3 configs
-func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, error) {
+func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, error) { //nolint:gocyclo
```

Comment:
@kron4eg I think it's okay to ignore this lint error because:

* this is a legacy branch
* this has been already fixed in 1.4 and master

WDYT?

Reply:
yeah, sounds good